.bazelrc

```diff
@@ -33,6 +33,14 @@ build:windows_arm64 --extra_toolchains=@local_config_cc//:cc-toolchain-arm64_win
 # Enable Bzlmod
 build:bzlmod --experimental_enable_bzlmod
 
+# Enable modern C++ features
+build:linux --cxxopt=-std=c++17
+build:linux --host_cxxopt=-std=c++17
+build:macos --cxxopt=-std=c++17
+build:macos --host_cxxopt=-std=c++17
+build:windows --cxxopt=/std:c++17
+build:windows --host_cxxopt=/std:c++17
+
```

Comment:
Does this imply that any other OSs do not need to have these settings?

Reply:
It's likely all OSs need those flags for building Bazel after upgrading abseil. But macOS, windows, linux basically covers all major platforms, right?

Reply:
It does. I was mostly wondering why we didn't go with the following:

```
build --cxxopt=/std:c++17
build --host_cxxopt=/std:c++17
```

Reply:
Oh, does `/std:c++17` also work on Unix system? I thought it has to be `-std:c++17`.